src/pages/Login/Login.scss

```diff
@@ -0,0 +1,66 @@
+@import "../../themes/colors.scss";
+@import "../../themes/theme.scss";
+
+.login-div {
+  padding: 1%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-family: $font-family-roboto;
+  p {
+    font-size: small;
+    color: grey;
+  }
+  .login-heading {
+    color: darkslateblue;
+    text-decoration: underline;
+  }
+  .nav-item {
+    width: inherit;
+  }
+  .nav-link {
+    color: $black;
+    text-decoration: none;
+  }
+  a.nav-link:hover {
+    color: $black !important;
+    font-weight: 600;
+    background-color: lightgrey;
+  }
+  a.nav-link.active {
+    color: $black !important;
+    font-weight: 600;
+    // background-color: lightgrey;
```

Comment:
please remove any and all unnecessary comments from the PR

Reply:
removed unnecessary comments